renv.lock

```diff
@@ -0,0 +1,4431 @@
+{
```

Comment:
I'm hesitant to use renv for the book - do we not always want it to be working with the latest versions of packages?